user-profiles.md

```diff
@@ -0,0 +1,49 @@
+
+## Preface
+
+* What's described here are user profiles as opposed to [personas](https://en.wikipedia.org/wiki/Persona#In_user_experience_design). Personas are example actors rather than general categories.
+* Kubernetes Cluster Operators are out of scope for this document. A cluster operator is one who manages the operation of a Kubernetes cluster that applications can run in.
+* Application Operators are in scope for this document at a lower priority than Pipeline Operators. An application operator is one who manages the operation of the application in a Kubernetes cluster.
+
+## Profiles
+
+Profiles describe a type of role a user may perform. A real person may perform more than one role and have more than one profile apply to them. How this mapping works between profiles and real people can vary between companies and other organizations. To handle this variation we focus on the user profiles rather than how they may map to people in these different organizations.
+
+### 1. Pipeline Operator
+
+Pipeline operators define, schedule, monitor, and otherwise operate pipelines across one or more Kubernetes clusters. For example, the operation of a set of Kaniko build pipelines and Argo CD pipelines. This is not to be confused with the role of a Kubernetes cluster operator. This is also not to be confused with the application's operator role, who would oversee a product in production.
+
+### 2. Pipeline Distributor
+
+Distributors are people who package an application and application pipelines for someone else to operate. Examples of this would be those who maintain [Catalog Pipelines](https://github.com/tektoncd/catalog) such as for Kaniko and ArgoCD.
+
+### 3. Application Operator and Application Distributor
+
+Application operators take an application and operate it within a Kubernetes cluster. For example, the operation of WordPress and MySQL. This is not to be confused with the role of a Kubernetes cluster operator (see earlier in preface for more detail). Application distributors are people who package an application for someone else to operate. Examples of this would be those who maintain community Helm Charts for WordPress and Datadog. These are the same profiles as seen in Helm's profile document https://github.com/helm/community/blob/master/user-profiles.md
+
+### 4. Application Developer
+
+An application developer writes the software for an application. They are typically not concerned with where the application is running as applications can often be run more than one environment (e.g., many applications can be run in a virtual machine or a container). Examples of this include the developers of WordPress and MySQL. These developers are also typically not concerned with what pipelines run to produce or ship an application, as long as the pipeline meets organizational needs (e.g., for signoff, security review, or other business workflows).
+
+### 5. Supporting Tool Developer
```

Comment:
Does this include "Platform Builders" creating products on Tekton?  I wonder if these users warrant their own profile. I would suggest that both the "Supporting Tool Developer" and hypothetical "Platform Builder" might be prioritized over Application Developers, especially given the Application Developer profile explicitly calls out that they're not concerned at all with Tekton as an implementation detail.

Reply:
Is a Platform Builder considered a Pipeline Distributor from Tekton's perspective? :thinking: Not that all of our decisions need to flow from Helm, though what would Helm consider a platform built on/around Helm charts? Or is there a set of additional tendrils a Platform Developer needs into Pipelines? Will consider moving the importance or clarifying importance of Platform Builders as causing Distribution of Pipelines above Application Developers. Though when Platform Builders wrap Tekton and hide all of its internal details, what is the motivation for Tekton to modify itself directly to meet the needs of layers above itself or hiding it?      I am not opposed to a change, just :thinking: a bit. Of course, Tekton wants to be a useful platform for development (right? *the* CICD platform for a K8S-native approach) and does not care about getting credit (generally speaking, per licensing). Helm also wants to be a useful platform, and defines that interface via its charting interface or system. I will place the platform builder at the #2 position currently, and may update text to reflect this after a bit more :thinking:

Reply:
I think Tekton currently aims for both being a useful platform in its own right as well as providing a common set of building blocks for other platforms to build with.  So I suggest giving those platform developers who are leveraging the building blocks an entry here since none of the other profiles really described their usage (as far as I could see when reading through).

Totally acceptable if this usage is described adequately by other profiles and I misread tho.

Reply:
I will first try to reword the entry for Pipeline Distributor to call out Platform Builder concerns directly. Failing that coming out smoothly, split it to a new user profile

Reply:
"Distributors are people who build platforms, including creating complex products, by packaging application pipelines for someone else to operate."

Reply:
I think I would prefer to see these split out. People building systems on top of Tekton (say Jenkins X for example) are going to have different concerns from people building/distributing Tasks and Pipelines.

Reply:
"Platform builders are people who wrap Tekton's core engine and libraries to extend it. An example of this is the platform being built by [Jenkins X](https://github.com/jenkins-x/jx) using Tekton pipelines."